enhancements/cluster-logging/cluster-logging-log-forwarding.md

```diff
@@ -35,53 +35,70 @@ The purpose of log forwarding is to provide a declarative way by which adopters
 container and node logs to destinations that are not necessarily managed by the OKD cluster logging infrastructure.
 Destinations are either on or off cluster endpoints such as the cluster logging provided Elasticsearch, an organization's Kafka message bus, or other log aggregation solutions (e.g. Splunk, Graylog).
 
+This document describes the initial release goals for log forwarding. "Future plans" at the end outlines stories that were pushed out during discussions.
```

Comment:
nit... consistency... "Log Forwarder".   maybe we should consider renaming the document

Reply:
I thought it was ok to use "forwarding" to describe the overall feature and "forwarder" to name the object that does forwarding - but I'm fine with using forwarder everywhere if you think it's clearer to rename.